### ISSUE TITLE
Update settings (pharmaverse - admiral packages)

### ISSUE THREAD

**Issue by dgrassellyb** [pharmaverse/admiralci#166]
1. Changing Pull requests conditions: 

Why: we just added some news r-cmd-checks on docker image - devel (docker image containing latest version of R 4.4 with latest packages). Admiral devs want to see these checks results, but without being blocked on daily devs and merge requests actions
How : 

Going on settings and branches : and edit protected branches rules : 

<img width="1561" alt="Screenshot 2023-11-07 at 15 49 55" src="https://github.com/pharmaverse/admiralci/assets/56442075/f9c72015-3a7e-43ed-9121-ba5adbbefa03">

Then check that Check / (devel) is not part of the required jobs that should pass : 

<img width="970" alt="Screenshot 2023-11-06 at 17 35 32" src="https://github.com/pharmaverse/admiralci/assets/56442075/f481df39-96f2-43e3-854b-443e8b8c7799">

2. Changing branch protection settings: 

Why : because we just added a new job (vbump) to automatically increase admiral maintenance versions on new changes on main branch (increasing vx.y.z.9001 to vx.y.z.9002 for example) 
How : 

First add `pharmaverse-bot` as admin in the repo.
Then on branch protection rules, unpick this option : 

![pharmaverse_bot_settings](https://github.com/pharmaverse/admiralci/assets/56442075/f8bbde96-a5d9-4e5f-9a61-f6f2db79edcb)

**Comment by bms63** [comment]
@dgrassellyb is this something that we need to do across all admiral packages to make use of the devel-image?  Can we just give you admin privileges across them all ??

**Comment by dgrassellyb** [comment]
> @dgrassellyb is this something that we need to do across all admiral packages to make use of the devel-image? Can we just give you admin privileges across them all ??

Hi @bms63, yes it would be nice to ensure this for every admiral packages, but as you said if I get admin rights I can do it, np !

**Comment by bms63** [comment]
> > @dgrassellyb is this something that we need to do across all admiral packages to make use of the devel-image? Can we just give you admin privileges across them all ??
> 
> Hi @bms63, yes it would be nice to ensure this for every admiral packages, but as you said if I get admin rights I can do it, np !

I updated you to admin on all the ones I have access to.  so admiralci, admiraldev, admiral, admiraltemplate.  might be good to get you admin rights on the data packages and admiral extension packages - maybe @manciniedoardo can set that up for you?

**Comment by rossfarrugia** [comment]
@dgrassellyb now given you admin access for admiralonco and admiralvaccine (fyi @neetusan @ahasoplakus)
 to make these updates for consistency across the family of packages.

**Comment by dgrassellyb** [comment]
(done)
